graphql/resolvers.go

```diff
@@ -1026,22 +1026,25 @@ func (r *queryResolver) MyHosts(ctx context.Context) ([]*restModel.APIHost, erro
 	return apiHosts, nil
 }
 
-func (r *queryResolver) ProjectSettings(ctx context.Context, identifier string) (*restModel.APIProjectSettings, error) {
+func (r *queryResolver) ProjectSettings(ctx context.Context, identifier string, isRepo bool) (*restModel.APIProjectSettings, error) {
```

Comment:
Would the workflow for this on the ui be: 
ProjectSettings = ProjectSettings(ctx, sandbox, false)
RepoSettings = ProjectSettings(ctx, sandbox, true)

instead of  doing it this way? 
ProjectSettings = ProjectSettings(ctx, sandbox)
RepoSettings = RepoSettings(ctx, sandbox)

Reply:
yep - we can make a RepoSettings one if you want instead but then we have to repeat all of the resolvers I'm assuming